src/main.rs

```diff
@@ -276,6 +278,69 @@ fn main() {
         }
     };
 
+    if args.edit_page && args.edit_patch {
+        print_error(
+            enable_styles,
+            &anyhow!("Don't edit-patch and edit-page at the same time"),
+        );
+        process::exit(1);
+    }
```

Comment:
This can done using clap's `conflicts_with` on one of the two options, for example
```rust
#[arg(long, conflicts_with = "edit_page")]
pub edit_patch: bool,
```